src/ig/IGExporter.ts

```diff
@@ -425,45 +426,78 @@ export class IGExporter {
       'vocabulary',
       'examples' // Must come last in case examples are of other resources
     ];
+    const predefinedFHIRDefs = new FHIRDefinitions();
     for (const pathEnd of pathEnds) {
+      let xmlFile = false;
+      let invalidFile = false;
       const dirPath = path.join(this.igDataPath, 'input', pathEnd);
       if (fs.existsSync(dirPath)) {
         const files = fs.readdirSync(dirPath);
         for (const file of files) {
           let resourceJSON: InstanceDefinition;
-          try {
-            resourceJSON = JSON.parse(fs.readFileSync(path.join(dirPath, file), 'utf-8').trim());
-          } catch (e) {
-            logger.error(`Invalid input file: ${path.join(dirPath, file)}`);
+          if (file.endsWith('.json')) {
+            resourceJSON = fs.readJSONSync(path.join(dirPath, file));
+          } else {
+            xmlFile = file.endsWith('.xml');
```

Comment:
This doesn't really aggregate the result, so `xmlFile` will always just be an indication of the last processed file.  I think you want it true as long as _any_ file was xml, right? If so, try:
```ts
xmlFile = xmlFile || file.endsWith('.xml');
```

Reply:
Ouch yeah fixed that here https://github.com/FHIR/sushi/pull/262/commits/7c3f17e30520a5f45ada23ce0b7e8f867904ed0f